.travis.yml

```diff
@@ -5,11 +5,25 @@ arch: arm64
 services:
   - docker
 jobs:
-  - env: RAKE_TASK="apt:build" APT_TARGETS="debian-buster"
-  - env: RAKE_TASK="apt:build" APT_TARGETS="ubuntu-bionic"
+  - env: RAKE_TASK="apt:build" APT_TARGETS="debian-buster" ARCHIVE_BASE="debian-buster" PACKAGE_SYSTEM=apt
+  - env: RAKE_TASK="apt:build" APT_TARGETS="ubuntu-bionic" ARCHIVE_BASE="ubuntu-bionic" PACKAGE_SYSTEM=apt
   # centos-release-scl isn't available for centos-7 on aarch64
   # TODO: Use rbenv instead
-  # - env: RAKE_TASK="yum:build" YUM_TARGETS="centos-7"
-  - env: RAKE_TASK="yum:build" YUM_TARGETS="centos-8"
+  # - env: RAKE_TASK="yum:build" YUM_TARGETS="centos-7" ARCHIVE_BASE="centos-7" PACKAGE_SYSTEM=yum
+  - env: RAKE_TASK="yum:build" YUM_TARGETS="centos-8" ARCHIVE_BASE="centos-8" PACKAGE_SYSTEM=yum
 script:
   - rake $RAKE_TASK
+before_deploy:
+  - export RELEASE_DIRECTORY="td-agent/${PACKAGE_SYSTEM}/repositories/"
+  - tar cJvf package-${ARCHIVE_BASE}-arm64.tar.xz td-agent/${PACKAGE_SYSTEM}/repositories
+  - echo "deploying tarball includes the following directory:"
+  - echo "${RELEASE_DIRECTORY}"
+deploy:
+  provider: releases
+  token:
+    secure: "JEr3rNHw9gNllQE1vkbO6rp4SH24UwDPs+wi1fMEgbOw/wbHlZg/E/xBGKZnb6/u8wUNweG6OJOeMlAFf+lqQWRmv03e0xu+GZqBdAxzJ+go9+J5WKcEbNz1VmWd0aqO+42kL5vJldZYSvlpEEqQxe1Ew6AbZ4tAhRP8rOV5yCcoprVIfU4szLxzHHGchMJ5ektwpU4oXCuRg/IBv9Uh6Zy5zEbeNWq4Mlvo8b1QWFgRH5LCDUF+GzhKOZOSH8C7OfGmh8lF2oeifBBWbaDTRBv5ysm2OfL/2A4O5t7bmRljsJody76i/8gA3bEkBay6RsFXVqbCeApxSiDWtLOeloXXbcUYGcg+139Ke2iwPKhCKEkYr37lhMfrZMJ29+1HbLo/4exQV81nTgEvTApHbe3TlJFatat442+JQv/E3q4Ses5nO2PqXSU5bgJ2VtSQG9yRMautcZ8/ns9VVS7hm03CK1nluJ5ltCbqriE7Sf+b0ieMN1inx1g8o4WtwUXB+ZS5S3nZWDRtsfrclvE2FVfphG/saR7KFcPUHe3P2UUEbYKiZhBei3oKYXuYT81KhrtvwIkSdySpuE2Tm33lnPwUWlPlOqK+Sd3ucofwTpO8lRq9AW6PBvU2MFANA238Jtl9K/deZOF6v8dIuUO/QJqlXokrvMbMmhyBi6eEKQ8="
```

Comment:
This secure token is taken with @hatake-buildbot .

Reply:
memo: It would be better that there is a mechanism to manage it by authorized users.